### PR TITLE
Add authoritative_source and distribution_scope labels

### PR DESCRIPTION
**Pull request by ghost**
Propsing two new fields to support Red Hat, ISV, and customer use cases after move to lowercase_underscore. Replaces https://github.com/projectatomic/ContainerApplicationGenericLabels/pull/23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,27 +17,39 @@ The following types of data are being considered:
...
+ | license     | Image License|
+ | build_date  | Date/Time image was built|
+ | vendor      | Owner of the image| 
+ | url         | Url with more information on the image|
+ | summary     | Short Description of the image|
+ | description | Detailed description of the image|
+ | vcs_type    | The type of version control used by the container source. Generally one of git, hg, svn, bzr, cvs|
+ | vcs_url     | URL of the version control repository|
+ | vcs_ref     | A 'reference' within the version control repository; e.g. a git commit, or a subversion branch|
+ | authoritative_source | The  authoritative registry in which the image is published|
```
Comment (jonboulle) [inline README.md:44]: Description needs an update

**Comment by rhatdan** [comment]
We changed the case, please rebase the authoritative source

**Comment by jmtd** [comment]
Note that since labels are case sensitive and the only way to remove a label in a downstream image is to re-define it, re-casing these variables can end up in a situation where downstream images may need/want to define them twice, pre and post rename, to ensure they mask out the ones in upstream images. if the upstream images haven't been relabelled/rebuilt.

**Comment by rhatdan** [comment]
Yes, I switched my tooling to look for exact match, Then lowercase, Uppercase, Capatized.

But the good thing is it really only pays attention to a few flags so far.

**Pull request closed without merging**